pyproject.toml

```diff
@@ -0,0 +1,104 @@
+[build-system]
+requires = ["flit_core >=3.11.0,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "pavics_sdi"
+requires-python = ">=3.10"
+authors = [
+  {name = "CRIM/Ouranos", email = "pavics@ouranos.ca"}
+]
+maintainers = [
+  {name = "Tuan Long Vu", email = "vu.long@ouranos.ca"},
+  {name = "David Huard", email = "huard.david@ouranos.ca"},
+  {name = "Trevor James Smith", email = "smith.trevorj@ouranos.ca"},
+  {name = "Francis Charette-Migneault", email = "francis.charette.migneault@gmail.com"}
```

Comment:
A gmail for Francis?  Did he explicitly asked for that? Shouldn't this be his crim.ca email?